interbtc-spec/docs/source/spec/introduction.rst

```diff
@@ -3,19 +3,26 @@ How to read this specification
 
 This specification is a living document. The actual implementation might deviate from the specification. In case of deviations in the code, the code has priority over the specification.
 
-Return types
```

Comment:
Can you leave in the return types please?

Reply:
This was Rust specific and we anyway describe return types in the postconditions.